### DISCUSSION TITLE
Duplicate issue, perhaps: Temperature conversion in the returned JSON field prior to 'hand-off'?

### DISCUSSION THREAD

**Discussion by discarn8** [grafana/grafana-json-datasource#269] [Q&A]
Sir:

At risk of duplicating the already asked questions in:
https://github.com/marcusolsson/grafana-json-datasource/issues/176
https://github.com/marcusolsson/grafana-json-datasource/issues/96

Is it possible to apply a mathematical conversion to a returned JSON field? I'm fetching the weather stats from https://fcc-weather-api.glitch.me/api/ and it returns the temperature in celsius. I've played with every known switch I can find on the grafana side to try field overrides, mapping, etc. but can't find anything that will allow me to conver that celsius to fahrenheit. By chance, is there a secret squirrel method in your plugin, of applying a conversion to the received data, prior to handing it off to grafana?

Thank you ahead of time and a sincere thanks for the awesome plugin.
~Rich

**Comment by marcusolsson** [comment]
If you use JSONata instead of JSONPath, you should be able to do basic math and other operations.

https://docs.jsonata.org/numeric-operators
